gcc/rust/checks/errors/rust-const-checker.cc

```diff
@@ -898,8 +901,12 @@ ConstChecker::visit (RawPointerType &)
 {}
 
 void
-ConstChecker::visit (ReferenceType &)
-{}
+ConstChecker::visit (ReferenceType &type)
+{
+  if (this->in_const_context && type.is_mut ())
+    rust_error_at (type.get_locus (),
+		   "Mutable references are not allowed in constant functions");
```

Comment:
```suggestion
		   "mutable references are not allowed in constant functions");
```

We usually go for lowercase error strings as it looks better when formatted by GCC

```shell
> gccrs
gccrs: fatal error: no input files
compilation terminated.
```